proto/tx.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
```

Comment:
The files in `proto` are just the implementation of LikeCoin chain, not any enforced interface for the service, so maybe better move into `sample` and give more explanation?